be/src/exec/spill/operator_mem_resource_manager.cpp

```diff
@@ -35,6 +35,9 @@ void OperatorMemoryResourceManager::to_low_memory_mode() {
         if (_spillable) {
             _query_spill_manager->increase_spilling_operators();
         }
+        if (_op->releaseable()) {
+            set_releasing();
+        }
```

Comment:
should it before L35?

Reply:
> should it before L35?

both are fine. since one opeartor cannot be spillable and releasable at the same time, for now